File: Maths/ArmstrongNum.cpp
// if n = 371, 3^3+7^3+1^3 = 371

#include <bits/stdc++.h>
using namespace std;
int main()
{
    int t = 2;
    while (t--)
    {
        int n, sum=0;
        cin >> n;
        int dup = n;
        while (n != 0)
        {
            int ld = n % 10;
            sum = sum + ld * ld * ld;
            n = n / 10;
        }
        if(dup == sum){
            cout << "Yes" << endl;
        }else{
            cout << "No" << endl;
        }
    }
}

File: Maths/ArmstrongNum.py
num = int(input("Enter the number: "))
sum_ = 0
temp = num 

while num!=0:
    ld = num%10
    # sum = sum + ld ** 3 
    # n = n/10
    sum_ += ld ** 3 
    num //= 10

if temp == sum_:
    print("Yes")
else:
    print("No") 